src/init.js

```diff
@@ -35,6 +35,7 @@ const Raven = require('raven-js');
     };
 
     window._locale = updateLocale();
+    document.documentElement.lang = window._locale;
```

Comment:
@rschamp just wanted you to have a look at this line

Reply:
Thanks — @LiFaytheGoblin what is the reason for this change?